storyblok/storyblok-design-system
# feature(int-512): datetime field date entry

In this PR we added the features requested by Nikola, they are:

- Enable the input for the user to type the date respecting the format imposed by the mask.

- Open the calendar by clicking on the calendar icon.

- Remove the clearable icon and allow the user to manually remove the date with the backspace.

- Add a specific style for the current date.

- Change the style of footer buttons and also change the text "Done" to "Apply".

- Add the `disabledPast` prop to disable dates before the current date.

- Add the `minDate` so that the user can enter a minimum viable date.

- Add the `maxDate` so that the user can enter a maximum viable date.

To make the above adjustments possible I also changed the SbTextField and added a new story 'With Mask'.

## Pull request type

Jira Link: [INT-512 - DateTime field manual date entry](https://storyblok.atlassian.net/browse/INT-512)




## How to test this PR
Open the datepicker in blok.ink and verify that the styling and new props are working correctly.
Also check out the new 'With Mask' story from SbTextField.



## Other information
Any questions, I'm available!
